packages/shared/src/enums.ts

```diff
@@ -30,6 +30,9 @@ export type ChainflipNetwork = (typeof ChainflipNetworks)[keyof typeof Chainflip
 export const isTestnet = (network: ChainflipNetwork): boolean =>
   network !== ChainflipNetworks.mainnet;
 
+export const isLocalnet = (network: ChainflipNetwork): boolean =>
+  network === ChainflipNetworks.backspin || network === ChainflipNetworks.sisyphos;
```

Comment:
sisyphos should undergo the runtime upgrade
```suggestion
  network === ChainflipNetworks.backspin;
```